docs/build/operator-class.ipynb

```diff
@@ -0,0 +1,745 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b88d9242-a6f2-47b6-8e0a-daf60d0c4374",
+   "metadata": {},
+   "source": [
+    "# The Operator class\n",
+    "\n",
+    "This page shows how to use the [`Operator`](/api/qiskit/qiskit.quantum_info.Operator) class. For a high-level overview of the class, see [Overview of operator classes](/build/operators-overview#operator)."
```

Comment:
```suggestion
    "This page shows how to use the [`Operator`](/api/qiskit/qiskit.quantum_info.Operator) class. For a high-level overview of the `quantum_info` module, including the `Operator` class and others, see [Operators module overview](/build/operators-overview)."
```
I think this is a bit clearer, currently it's a bit vague to me about what is the difference between the 2 pages just from the intro description

Reply:
That page doesn't actually give an overview of the `quantum_info` module, only the operator classes. I've changed the sentence to

>For a high-level overview of operator representations in Qiskit, including the `Operator` class and others, see [Overview of operator classes](/build/operators-overview).